server/openai/config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  openaiApiKey: process.env.OPENAI_API_KEY || 'fallback_openai_api_key_here',
```

Comment:
do we want to have a fallback key committed in cleartext? what would we put here?

Reply:
mainly had it for local dev but I can change this

Reply:
oh I thought we would still use our own key that's written in .env for dev, and the deployment would have its own .env file. 

feel free to keep the fallback key if it has a use, though otherwise let's not commit keys :)